lib/ansible/module_utils/facts/hardware/linux.py

```diff
@@ -443,25 +443,13 @@ def _mtab_entries(self):
             mtab_entries.append(fields)
         return mtab_entries
 
-    @timeout.timeout()
-    def get_mount_facts(self):
-        mount_facts = {}
-
-        mount_facts['mounts'] = []
+    def get_mount_info(self, fields, bind_mounts, uuids):
```

Comment:
This can become a function since it no longer makes use of self.

Reply:
has still 2 uses of self